Assignment2/Question1.java

```diff
@@ -0,0 +1,138 @@
+package Assignment2;
+
+import java.util.Stack;

```

Comment:
FYI: Stack extends Vector, and Vector is not really encouraged for use in Java anymore. ArrayList is much preferred.